### ISSUE TITLE
Creating a healthbargump limit and decay timer.

### ISSUE THREAD

**Issue by Alias-Alan** [ClassicUO/ClassicUO#1196]
**Describe the bug**
Due to the size of custom healthbars, people typically pvpers can be seen with over 15 health bars(up to 50 at times) and often times they are healthbars pulled from prior gaming sessions.

**Expected behavior**
Healthbars are expected to decay/dispose and not be used to store an absurd amount of them at one time.

**Shard**
All - this was an unintended consequence that was overlooked when I originally developed these.

**Possible resolution**
A bool to indicate an active/inactive healthbar assigned to a timer regarding said healthbars last seen in range of World.Player. A timer would account for disconnects but limited to ~4hrs.
A reasonable hard cap for the amount of health bars able to be created(similiar to the original client).

**Comment by andreakarasho** [comment]
There are some problems/tips with this request:
- i should maintain a timer based on what? 
- shard can handle healthbars and close them if necessary
- this will go in contrast to the "keep healthbars when logout" option

Original client keeps 10 healthbars only. If you grab the 11th it will remove the latest you added.
Is that worth? What's the issue and how keeping 1 or 100 healthbars in the same time affects the gameplay?

**Comment by Alias-Alan** [comment]
Well, when creating those custom healthbars the original idea was to be able to have it setup to use default or use your own custom provided it met certain specifications such as Length and Height and an empty transparent space to place the actual health bar line. It never crossed my mind that people would maintain 40+ health bars on their screen. 

As much as you've wanted to recreate an original feel to a modern design I understood why my original idea was not one you would approve. Sallos obviously had sleek health bars though a little smaller with less empty space. They were the wheel I reinvented and believe I improved with the colored backgrounds, the ability to know which target is your lasttarget by the red outline if their health bar. I even have other features we can discuss outside of this to stay on track.

Due to the size of the health bars I think if 10 was the original maximum, 20 is a reasonable compromise. However, at present and aforementioned, people have 40+ on their screen. They can log out and have those same health bars days later.

I thought the feature to save health bars on disconnect/logout was a clever qol feature. Unfortunately, much like many other things, such a feature is prone to being abused and takes away a part of gameplay that adds to the challenge of it. My decay timers is a way to preserve the qol feature while also limiting its abuse potential.
